src/app/components/projects/pages/harvest/harvest.component.ts

```diff
@@ -1,143 +1,45 @@
-import { Component, OnInit, ViewEncapsulation } from "@angular/core";
+import { Component, OnInit } from "@angular/core";
 import { ActivatedRoute } from "@angular/router";
 import { projectResolvers } from "@baw-api/project/projects.service";
 import { PageComponent } from "@helpers/page/pageComponent";
 import { permissionsWidgetMenuItem } from "@menu/widget.menus";
 import { Project } from "@models/Project";
-import { ResolvedModel } from "@services/baw-api/resolver-common";
-import filesize from "filesize";
 import { List } from "immutable";
-import { Observable, Subscription, timer } from "rxjs";
-import { map, startWith, takeUntil, takeWhile } from "rxjs/operators";
 import { harvestProjectMenuItem, projectCategory } from "../../projects.menus";
 import { projectMenuItemActions } from "../details/details.component";
 
+export enum HarvestStage {
+  start,
+  streamingUpload,
+  batchUpload,
+  uploadVerification,
+  uploadReview,
+  fileVerification,
+  fileReview,
+  complete,
```

Comment:
I'd go through effort to ensure these match the statuses that could be returned from the server. https://github.com/QutEcoacoustics/baw-server/compare/harvest-table?diff=unified#diff-7ba7cc9ed72155500e84a719fb6214316f1fe58b7537ad27324d6b935c0a8f0cR75

Reply:
So you might implement the actual decisions on which component to show based on a predicate.

```
start: h => h == null,
streamingUpload: h => h.streaming && h.status == Uploading
batchUpload: h => !h.streaming && h.status == Uploading
```

etc..